### ISSUE TITLE
Error when retrieving fields of type DateTime

### ISSUE THREAD

**Issue by patrobinson** [cleishm/libneo4j-client#36]
When I try and query a Node that has a DateTime property I get the following error:
```
<interactive>:0:0: error: DateTime is not supported as a return type in Bolt protocol version 1. Please make sure driver supports at least protocol version 2. Driver upgrade is most likely required.
```

I'm running the latest version
```
11:35 $ neo4j-client --version
neo4j-client: 2.2.0
libneo4j-client: 2.2.0
libcypher-parser: 0.6.0
```

**Comment by cleishm** [comment]
Right - the latest Neo4j version has made a non-documented or standardized protocol upgrade. There is some work in the https://github.com/cleishm/libneo4j-client/tree/neo4j-proto-2 branch to reverse engineer the additions and add support for them.

**Comment by hga** [comment]
Is libneo4j-client effectively dead with Bolt now having moved to a further and also undocumented v3 of the wire protocol?

**Comment by cleishm** [comment]
Not necessarily. V1 bolt works ok, and it should not be too hard to reverse engineer the protocol given the server implementation is open source. It just needs a someone to do so.

**Comment by cleishm** [comment]
@seajaysec ran into the same issue. Is anyone interested in contributing to this? @agatlin?
